src/main/java/net/minecraftforge/common/world/ForgeChunkManager.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Minecraft Forge
- * Copyright (c) 2016-2021.
+ * Copyright (c) 2016-2020.
```

Comment:
Why are you changing this?

Reply:
This is the work of the `updateLicenses` task, which checks that license headers conforms to the one specified in `LICENSE-headers.txt` and if not, replaces them. Without this, the `checkLicenses` task will complain.

I'd say that this change be okayed for now, until someone can update the license header file and all of the license headers via `updateLicenses` in another PR, to maintain consistency and avoid `checkLicenses` from erroring.